extra/generate-display-names/display-names

```diff
@@ -103,6 +103,7 @@
   ../../tables/IPA.utb                              International Phonetic Alphabet                    International Phonetic Alphabet braille
 * ../../tables/is.tbl                               Icelandic                                          Icelandic braille
 * ../../tables/it.tbl                               Italian                                            Italian braille
+* ../../tables/it-it-comp6.utb                      Italian, computer, 6-dot                           Italian 6-dot computer braille
```

Comment:
This table was already included as it.tbl ("Italian").

Reply:
It would be good to delete it.tbl and move the metadata, but it should probably stay consistent.